package.json

```diff
@@ -72,13 +73,14 @@
     "crypto-browserify": "^3.12.0",
     "google-protobuf": "^3.12.2",
     "grpc-web": "1.2.1",
+    "isomorphic-ws": "^4.0.1",
```

Comment:
Switched to `isomorphic-ws` for browser compatibility (we want to use `WebSocket` in the browser instead of `ws`).

Reply:
sounds like we definitely want to go this route since xpring-js needs to be browser compatible